Doc/library/getopt.rst

```diff
@@ -120,7 +124,9 @@ Using long option names is equally easy:
    >>> args
    ['a1', 'a2']
 
-In a script, typical usage is something like this::
+In a script, typical usage is something like this:
+
+.. testcode::
```

Comment:
Is it needed? There is no test output. Also, the code is not self-sufficient -- it refers to not defined functions.